src/angular/styles/_typography.scss

```diff
@@ -436,8 +436,7 @@ p,
 // Link
 // ----------------------------------------------------------------------------------------------------
 
-a:not(.sbb-menu-item),
-.sbb-link:not(.sbb-menu-item) {
+.sbb-link {
```

Comment:
Links are not displayed with correct styles anymore, e.g. in introduction or sbb-header mobile panel. I think it's a bad solution to not provide styles for the a tag in typography.

Reply:
Let's discuss this via Teams. I do have a reason for this, which I forgot to discuss with you yesterday.